extensions/amp-skimlinks/0.1/skim-options.js

```diff
@@ -147,6 +150,29 @@ function getInternalDomains_(docInfo) {
   return internalDomains;
 }
 
+/**
+ *
+ * @param {!Element} element
+ * @return {string}
+ */
+function getWaypointBaseUrl(element) {
+  let customSubDomain = element.getAttribute('custom-redirect-domain');
+  if (customSubDomain) {
+    // Remove potential HTTP protocol
+    customSubDomain = customSubDomain.replace(/^\/\/|^https?:\/\//, '');
+    // Remove potential trailing slash
+    customSubDomain = endsWith(customSubDomain, '/') ?
```

Comment:
this doesn't seem to handle the case of custom.subdomain.com/beer
not sure if its a realistic concern tho